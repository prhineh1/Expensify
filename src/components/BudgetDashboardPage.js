import React from 'react';
import BudgetsSummary from './BudgetsSummary';
import BudgetList from './BudgetList';

const BudgetDashboardPage = () => (
  <div>
      <BudgetsSummary />
      <BudgetList />
  </div>
);

export default BudgetDashboardPage;