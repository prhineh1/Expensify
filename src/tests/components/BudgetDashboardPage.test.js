import React from 'react';
import { shallow } from 'enzyme';
import BudgetDashboardPage from '../../components/BudgetDashboardPage';

test('should render BudgetDashboardPage', () => {
  const wrapper = shallow(<BudgetDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});