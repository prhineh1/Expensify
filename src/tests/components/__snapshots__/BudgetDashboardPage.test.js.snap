// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render BudgetDashboardPage 1`] = `
<div>
  <BudgetsSummary />
  <Connect(BudgetList) />
</div>
`;
